receiver/k8sclusterreceiver/watcher.go

```diff
@@ -45,6 +46,8 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8sclusterreceiver/internal/utils"
 )
 
+const clusterWideInformerKey = ""
```

Comment:
Maybe it would be more straight-forward to use key like `<cluster-wide-informer-key>` or sth?

Reply:
good point, something like that might be easier to understand - will adapt that